build.js

```diff
@@ -429,36 +429,46 @@ const getExtAttr = (node, name) => {
   return node.extAttrs && node.extAttrs.find((i) => i.name === name);
 };
 
-// https://webidl.spec.whatwg.org/#dfn-exposure-set
+// https://webidl.spec.whatwg.org/#Exposed
 const getExposureSet = (node) => {
-  // step 6-8
+  // step 6-8 of https://webidl.spec.whatwg.org/#dfn-exposure-set
   const attr = getExtAttr(node, 'Exposed');
   if (!attr) {
     throw new Error(
       `Exposed extended attribute not found on ${node.type} ${node.name}`
     );
   }
-  const globals = new Set();
+  const exposure = new Set();
   switch (attr.rhs.type) {
     case 'identifier':
-      globals.add(attr.rhs.value);
+      exposure.add(attr.rhs.value);
       break;
     case 'identifier-list':
       for (const {value} of attr.rhs.value) {
-        globals.add(value);
+        exposure.add(value);
+      }
+      break;
+    case '*':
+      for (const value of [
+        'Window',
```

Comment:
Per spec this should be the set of all interfaces which have the `[Global]` extended attribute. That includes https://webaudio.github.io/web-audio-api/#audioworkletglobalscope and other things not on this list. We don't generate any tests for worklets yet, but I think we should still collect all of the valid global scopes from parsing all of the IDL. We'd use that set here, but we can also use it to validate that there aren't any typos in global scope names anywhere.

Reply:
Just got that logic added in to grab everything with a `[Global]` extended attribute, with a bit of remapping, and use that as the value for wildcard exposures.  I've also added in the validation logic for an extra touch!